cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -932,13 +1027,13 @@ private BasicStats basicStats() {
     private BasicStats(double diskUtil, double cpuUtil, double leaderBytesInRate,
                        double followerBytesInRate, double bytesOutRate, double potentialBytesOutRate,
                        int numReplicas) {
-      _diskUtil = diskUtil;
-      _cpuUtil = cpuUtil;
-      _leaderBytesInRate = leaderBytesInRate;
-      _followerBytesInRate = followerBytesInRate;
-      _bytesOutRate = bytesOutRate;
-      _potentialBytesOutRate = potentialBytesOutRate;
-      _numReplicas = numReplicas;
+      _diskUtil = Double.isNaN(diskUtil) || diskUtil < 0.0 ? 0.0 : diskUtil;
```

Comment:
After #50 we should not get NaN any more.

Reply:
removing isNaN check